/**:
  active_command_node:
    ros__parameters:
      use_sdk: false
      sdk_max_speed: 3.0
      turn_max_speed: 6.0
      pitch_max_pose: 1.0
      roll_max_pose: 1.0
      height_max_pose: 1.0
      pub_freq: 170.0
      speed_max_ratio: 1.0
      speed_mid_ratio: 0.5
      speed_min_ratio: 0.15
      height_max: 0.3
      height_mid: 0.2
      height_min: 0.1
      